Esri/arcgis-maps-sdk-swift-samples
# [New] Show realistic light and shadows

## Description

This PR implements `Show realistic light and shadows` in `Scenes ` category.
URL to README: [URL](https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/des12437/Realistic-lighting-and-shadows/Shared/Samples/Realistic%20lighting%20and%20shadows/README.md)

## Linked Issue(s)

- `swift/issues/3859`

## How To Test
- Use the slider to adjust time of day from 12 AM to 11:59 PM.
- Tap the `Mode` button to trigger the lighting modes popup.
- Tap a mode on the popup to see the scene use the selected lighting mode.

## Screenshots
![realistic-lighting-and-shadows](https://user-images.githubusercontent.com/117859673/229248465-f98b181b-a436-43d3-a3fb-4c79ef3aeb6f.png)
